addons/l10n_ae_pos/__manifest__.py

```diff
@@ -9,7 +9,11 @@
 =======================================================
     """,
     'depends': ['l10n_ae', 'point_of_sale'],
-    'qweb': ['static/src/xml/Screens/ReceiptScreen/OrderReceipt.xml'],
     'auto_install': True,
     'license': 'LGPL-3',
+    'assets': {
+        'web.assets_qweb': [
+            'l10n_ae_pos/static/src/xml/Screens/ReceiptScreen/OrderReceipt.xml',
```

Comment:
was it working before ? because if this file wasn't integrated to the templates, there is a test missing !

Reply:
People didn't bother with tests for localisation before it was integrated in the CI (since 15.0)

Reply:
put it in 'point_of_sale.assets'